java/src/main/java/com/logicalclocks/hsfs/engine/FeatureViewEngine.java

```diff
@@ -170,43 +167,44 @@ public TrainingDatasetBundle getTrainingDataset(
             String.format("Incorrect `get` method is used. Use `FeatureView.%s` instead.", methodName));
       }
     }
-    if (!IN_MEMORY_TRAINING_DATASET.equals(trainingDatasetUpdated.getTrainingDatasetType())) {
-      try {
-        List<Split> splits = trainingDatasetUpdated.getSplits();
-        if (splits != null && !splits.isEmpty()) {
-          Map<String, Dataset<Row>> datasets = Maps.newHashMap();
-          for (Split split : splits) {
-            datasets.put(split.getName(),
-                trainingDatasetEngine.read(trainingDatasetUpdated, split.getName(), userReadOptions));
-          }
-          return new TrainingDatasetBundle(trainingDatasetUpdated.getVersion(),
-              datasets, featureView.getLabels());
-        } else {
-          return new TrainingDatasetBundle(trainingDatasetUpdated.getVersion(),
-              trainingDatasetEngine.read(trainingDatasetUpdated, "", userReadOptions), featureView.getLabels()
-          );
-        }
-      } catch (InvalidInputException e) {
-        throw new IllegalStateException(
-            "Failed to read datasets. Check if path exists or recreate a training dataset."
-        );
-      }
-    } else {
-      Dataset<Row> dataset = readDataset(featureView, trainingDatasetUpdated, userReadOptions);
-      TrainingDatasetBundle trainingDatasetBundle;
-      if (trainingDatasetUpdated.getSplits() != null && !trainingDatasetUpdated.getSplits().isEmpty()) {
-        Dataset<Row>[] datasets = SparkEngine.getInstance().splitDataset(trainingDatasetUpdated, dataset);
-        trainingDatasetBundle = new TrainingDatasetBundle(trainingDatasetUpdated.getVersion(),
-            convertSplitDatasetsToMap(trainingDatasetUpdated.getSplits(), datasets),
-            featureView.getLabels());
-        computeStatistics(featureView, trainingDatasetUpdated, datasets);
-      } else {
-        trainingDatasetBundle = new TrainingDatasetBundle(trainingDatasetUpdated.getVersion(), dataset,
-            featureView.getLabels());
-        computeStatistics(featureView, trainingDatasetUpdated, new Dataset[]{dataset});
-      }
-      return trainingDatasetBundle;
-    }
+//    if (!IN_MEMORY_TRAINING_DATASET.equals(trainingDatasetUpdated.getTrainingDatasetType())) {
```

Comment:
why did it have to be commented out? why not just remove it if it is not necessary?